core/src/main/res/values/themes.xml

```diff
@@ -2,38 +2,45 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
 
   <!--Top level theme to be used in AndroidManifest.xml-->
-  <style name="KiwixTheme" parent="Base.KiwixTheme" />
+  <style name="KiwixTheme" parent="Base.KiwixTheme">
+
+    <!--colorPrimary colors map to components and elements, such as app bars and buttons. -->
+    <!--colorSecondary colors are most often used as accents on components, such as FABs and -->
```

Comment:
these comments are a bit disjointed, should be moved and probably reworded a bit